mock-core-configs/mock-core-configs.spec

```diff
@@ -23,21 +26,22 @@ Requires(post): coreutils
 %if 0%{?fedora} > 29 || 0%{?rhel} > 8
 BuildRequires:  systemd-rpm-macros
 %endif
-%if 0%{?fedora} || 0%{?mageia} || 0%{?rhel} > 7
 # to detect correct default.cfg
-Requires(post): python3-dnf
-Requires(post): python3-hawkey
-Requires(post): system-release
-Requires(post): python3
-Requires(post): sed
-%endif
-%if 0%{?rhel} && 0%{?rhel} <= 7
+%if 0%{?rhel} > 0 && 0%{?rhel} <= 7
```

Comment:
This should be `< 7` otherwise this PR does not have the sense at all, because for RHEL 8 `python3_pkgversion` is simply 3. It is only RHEL7 where it is clumsy and `python3_pkgversion` is 36.